### PR TITLE
Added OxygenSaturation samples

### PR DESCRIPTION
**Pull request by buzzeh**
Added the ability to get the Oxygen Saturation samples

### CODE REVIEW

**Comment by joaovpmamede** [comment]
I was just waiting for this.
Any chance to include this in the next release @lucaspbordignon ?

**Review by lucaspbordignon — DISMISSED** [review]

**Review by Macelai — CHANGES_REQUESTED** [review]

**Comment by lucaspbordignon** [comment]
Thanks, @buzzeh

That one is a valuable addition to the project, considering the recent addition of the OxygenSaturation in Apple HealthKit

Will definitively include on the next release of the library (@joaovpmamede FYI)

File: RCTAppleHealthKit/RCTAppleHealthKit+TypesAndPermissions.m [hunk 1]
```diff
@@ -88,6 +88,9 @@ - (nullable HKObjectType *)getReadPermFromText:(nonnull NSString*)key {
     } else if ([@"RespiratoryRate" isEqualToString: key]) {
         return [HKObjectType quantityTypeForIdentifier:HKQuantityTypeIdentifierRespiratoryRate];
     }
+    else if ([@"OxygenSaturation" isEqualToString: key]) {
```
[the remark this answers is not in the document]
Comment (Macelai) [reply RCTAppleHealthKit/RCTAppleHealthKit+TypesAndPermissions.m:91]: Just review this one, please

**Review by Macelai — CHANGES_REQUESTED** [review]

File: RCTAppleHealthKit/RCTAppleHealthKit+TypesAndPermissions.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (buzzeh) [reply RCTAppleHealthKit/RCTAppleHealthKit+TypesAndPermissions.m]: Let me know if that is all, this is my first pull request, so I apologize if Im not doing it right

**Review by lucaspbordignon — APPROVED** [review]

**Comment by Macelai** [comment]
That’s perfect! Thanks for the quick reply on that one 😄

**Review by Macelai — APPROVED** [review]

**Comment by lucaspbordignon** [comment]
Released at version 1.5.0

@joaovpmamede FYI

**Pull request merged**